employees/tests/test_report_list_views.py

```diff
@@ -432,6 +498,18 @@ def test_custom_report_list_view_should_display_monthly_hour_sum_in_hour_format(
         self.assertContains(response, "32:00")
         self.assertNotContains(response, str(timezone.timedelta(hours=32)))
 
+    def test_custom_report_list_view_should_not_include_expected_hours_from_another_month(self):
+        other_expected_hours = ExpectedHoursFactory(
+            user=self.user,
+            project=self.project,
+            date=timezone.datetime(year=self.current_time.year, month=self.current_time.month + 1, day=1),
+            value=timedelta(hours=4),
+        )
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        sum_expected_hours = response.context_data["expected_hours_monthly_sum"]
+        self.assertNotEqual(sum_expected_hours, other_expected_hours.value + self.expected_hours.value)
```

Comment:
It's always better to use `assertEqual` because you are sure that you have value which you expect you to have. This statement which you have here will be true when it will be 8 expected hours as well as 5901237841902741290 hours.

Reply:
Changed